src/components/Contacts/ContactDetails/ContactDetailsHeader/ContactDetailsHeader.tsx

```diff
@@ -1,21 +1,29 @@
 import { Box } from '@material-ui/core';
 import React from 'react';
 
-import { ContactDetailsFragment } from '../ContactDetails.generated';
+import { useGetContactDetailsHeaderQuery } from './ContactDetailsHeader.generated';
 
 interface Props {
-  loading: boolean;
-  contact?: ContactDetailsFragment;
+  accountListId: string;
+  contactId: string;
 }
 
 export const ContactDetailsHeader: React.FC<Props> = ({
-  loading,
-  contact,
+  accountListId,
+  contactId,
 }: Props) => {
+  const { data, loading } = useGetContactDetailsHeaderQuery({
+    variables: { accountListId, contactId },
+  });
+
   return (
     <Box>
       {/*TODO: Build Header*/}
-      {loading ? <p>loading</p> : <p role="contactName">{contact?.name}</p>}
+      {loading ? (
+        <p>loading</p>
```

Comment:
I assume this loading text is just placeholder? I've been using the ```Skeleton``` component for loading states and it visually works very well. Here's an example: https://github.com/CruGlobal/mpdx-react/blob/54d75ad1c29cc1a2f6670de4200be234d2b503f8/src/components/Dashboard/DonationHistories/DonationHistories.tsx#L172-L176

Reply:
Material UI does provide a Indeterminate Progress Indicator that might be useful to use for loading UIs.

https://material-ui.com/components/progress/#circular-indeterminate

Reply:
Thanks!  I might use Skeleton for now, since we've used that in other places already.  But yes, this is just a placeholder, I'm actually already working on the header in another branch.  Maybe I can leave this as it is for now, and create a proper loading state in that other branch?

Reply:
Doing it in another branch is fine with me 🙂